### PR TITLE
[TW-383] Audit logs update

### PR DESCRIPTION
**Pull request by claireriley**
Updated + alphabetized audit logs

### CODE REVIEW

File: src/pages/docs/administration/audit-logs.md [hunk 1]
```diff
@@ -67,32 +66,68 @@ Logged events include the following actions.
...
 | Added Member   | A user joined your team.   |
+| Added Payment Method  | A user added a payment method. |
+| Approved Invite | An admin approved an email invitation. |
+| Approved join request | An admin approved a join request to team.
 | Cancelled Invite   | An invitation for a user was cancelled.   |
-| Successful login   | A user logged in successfully.   |
-| Logout | A user logged out. |
-| Reset Password | A user reset their password. |
+| Cancelled Invoice | A user cancelled an invoice for your team. |
+| Cancelled Plan | A user cancelled team’s subscription. |
```
Comment (HeatherStenson) [inline src/pages/docs/administration/audit-logs.md:81]:
```suggestion
| Cancelled Plan | A user cancelled the team’s subscription. |
```

Comment (HeatherStenson) [reply src/pages/docs/administration/audit-logs.md:81]: Or maybe `your team's`, based on the line above? 🤔

File: src/pages/docs/administration/audit-logs.md [hunk 2]
```diff
@@ -67,32 +66,68 @@ Logged events include the following actions.
...
+| Team URL updated | Your team’s URL was changed. |
+| Updated Billing Email | A user changed your team's billing email address. |
+| Updated Custom Domain Verification Status | The verification status of your domain was updated. |
+| Updated Group | A user updated a group. |
+| Updated Invoice Details | A user updated invoice details. |
+| Updated Monitoring Block Count | A user updated monitoring block count. |
+| Updated Team Discovery | A user updated your team's discoverability status. |
+| Updated User Role | A user updated a user role. |
+| Updated User Roles | Roles were updated for some users in your team. |
+| User Joined Via Domain Capture | A user joined through team domain management |
```
Comment (HeatherStenson) [inline src/pages/docs/administration/audit-logs.md:130]:
```suggestion
| User Joined Via Domain Capture | A user joined through [team domain management](/docs/administration/managing-your-team/configuring-domain-capture/). |
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

File: src/pages/docs/administration/audit-logs.md [hunk 3]
```diff
@@ -67,32 +66,68 @@ Logged events include the following actions.
...
 | Action name  | Description |
 | ------------- | ------------- |
-| Added Payment Method  | A new credit card was added to the your team.  |
-| Removed Payment method   | A credit card was removed from your team.  |
-| Added Domain   | A custom domain was added to your team. (You can use custom domains to publish documentation.)  |
-| Deleted Domain  | A custom domain was deleted from your team.  |
+| Activated SCIM Key | A user activated an SCIM key. |
+| Added Custom Alert | A new [custom token alert](/docs/api-security/token-scanner/#custom-alerts) was created for your team. |
+| Added Custom Domain | A custom domain was added to your team. (You can use custom domains to publish documentation.)  |
+| Added Domain For Domain Capture | Team admin added domain for domain capture. |
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/audit-logs.md:72]:
```suggestion
| Added Domain For Domain Capture | A team admin added a domain for domain capture. |
```

File: src/pages/docs/administration/audit-logs.md [hunk 4]
```diff
@@ -67,32 +66,68 @@ Logged events include the following actions.
...
+| Added Payment Method  | A user added a payment method. |
+| Approved Invite | An admin approved an email invitation. |
+| Approved join request | An admin approved a join request to team.
 | Cancelled Invite   | An invitation for a user was cancelled.   |
-| Successful login   | A user logged in successfully.   |
-| Logout | A user logged out. |
-| Reset Password | A user reset their password. |
+| Cancelled Invoice | A user cancelled an invoice for your team. |
+| Cancelled Plan | A user cancelled team’s subscription. |
+| Changed Default Payment Method | A user changed default payment method. |
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/audit-logs.md:82]:
```suggestion
| Changed Default Payment Method | A user changed the default payment method. |
```

File: src/pages/docs/administration/audit-logs.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/administration/audit-logs.md:72]: Per Heather's comments, it seems like most of the entries use articles, so good to be consistent 🙂

File: src/pages/docs/administration/audit-logs.md [hunk 5]
```diff
@@ -67,32 +66,68 @@ Logged events include the following actions.
...
 | Changed Password | A user changed their password. |
-| Custom auth scheme created| A new SSO scheme was added to your team.  |
+| Changed Plan | A user changed team plan. |
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/audit-logs.md:84]:
```suggestion
| Changed Plan | A user changed the team plan. |
```

File: src/pages/docs/administration/audit-logs.md [hunk 6]
```diff
@@ -67,32 +66,68 @@ Logged events include the following actions.
...
 | Custom auth scheme enabled | An SSO scheme was enabled.  |
 | Custom auth scheme removed  | An SSO scheme was removed.  |
-| Custom auth scheme updated  | An SSO scheme was updated.|
+| Custom auth scheme updated  | An SSO scheme was updated. |
 | Decreased Team Size  | Extra licenses were removed from the team. |
+| Deleted Custom Alert  | A custom token alert was deleted. |
+| Deleted Custom Domain  | A custom domain was deleted from your team. |
+| Deleted Domain For Domain Capture | Team admin deleted a domain for domain capture. |
+| Deleted Group | A user deleted a group. |
+| Deleted Payment Method | A user deleted payment method. |
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/audit-logs.md:96]:
```suggestion
| Deleted Payment Method | A user deleted a payment method. |
```

File: src/pages/docs/administration/audit-logs.md [hunk 7]
```diff
@@ -67,32 +66,68 @@ Logged events include the following actions.
...
+| Deleted SCIM Key | A user deleted an SCIM key. |
+| Disabled Domain Team Management | Team admin disabled domain team management. |
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/audit-logs.md:98]:
```suggestion
| Disabled Domain Team Management | A team admin disabled domain team management. |
```

File: src/pages/docs/administration/audit-logs.md [hunk 8]
```diff
@@ -67,32 +66,68 @@ Logged events include the following actions.
...
+| Downloaded Exported Data | A user downloaded an export of their data. |
+| Edited Custom Alert | A custom token alert's name or regex pattern was changed, or the alert was turned on or off. |
+| Enabled Domain Team Management | Team admin enabled domain team management. |
+| Exported Data | A user requested an export of their data. |
 | Increased Team Size | Additional licenses were added to the team.  |
+| Regenerated SCIM Key | A user regenerated an SCIM key. |
+| Rejected Invite | An admin rejected an email invitation. |
+| Rejected Join Request | A user rejected a join request to team. |
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/audit-logs.md:106]:
```suggestion
| Rejected Join Request | A user rejected a join request to the team. |
```

File: src/pages/docs/administration/audit-logs.md [hunk 9]
```diff
@@ -67,32 +66,68 @@ Logged events include the following actions.
...
+| Removed Group Member | A user is removed from the group. |
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/audit-logs.md:107]:
```suggestion
| Removed Group Member | A user was removed from the group. |
```

**Review by mark-anthony-postman — APPROVED** [review]
Some super minor suggestions, otherwise LGTM! 🙂

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Pull request merged**